src/manipulate.jl

```diff
@@ -35,8 +43,11 @@ macro manipulate(expr)
         bindings = [expr.args[1]]
     end
     syms = symbols(bindings)
-    Expr(:let, Expr(:block,
-                    display_widgets(syms)...,
-                    esc(map_block(block, syms))),
-         map(make_widget, bindings)...)
+    Expr(:let,
+            Expr(:tuple,
+                map(sym->esc(sym), syms)...,
+                esc(Expr(:...,map_block(block, syms)))
+            ),
+            map(make_widget, bindings)...
+    )
```

Comment:
Why not make this a composite type instead of a tuple? Overriding methods on `Vararg{IType}` looks hacky to me. Besides, that means we can't have anything other than IType things in the output.

Something like:

```julia
immutable Manipulate
    widgets::Dict # parameter name to widget mapping
    output::Signal
    layout::Function # Manipulate -> hbox / vbox (Can layout widgets using hbox and vbox) - has default, can be configued via @manipulate if necessary
end
```

could do the trick. Is it possible now to display _signals_ in an `hbox` or `vbox`? Then the default layout could just `vbox` the output below the widgets... Otherwise for now we could just return the output and let IJulia handle it as it now does.